<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>check</class>
 <widget class="QMainWindow" name="check">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(103, 59, 99);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_like_teacher">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 166, 15);</string>
    </property>
    <property name="text">
     <string>Зарегистрироваться как учитель</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_like_student">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 166, 15);</string>
    </property>
    <property name="text">
     <string>Зарегистрироваться как ученик</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>247</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 166, 15);</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <zorder>btn_like_student</zorder>
   <zorder>btn_like_teacher</zorder>
   <zorder>btn_back</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
